config.py

```diff
@@ -64,6 +64,7 @@ class Config(object):
     # Feature Flags
     RAISE_ERROR_ON_MISSING_FEATURES = True
     FEATURE_FLAGS_NEW_SUPPLIER_FLOW = False
+    FEATURE_FLAGS_DIRECT_AWARD = False
```

Comment:
Would DIRECT_AWARD_PROJECTS be more clear? We can't enabling/disabling G-Cloud after all :)